subsys/bluetooth/audio/Kconfig.mpl

```diff
@@ -0,0 +1,115 @@
+# Bluetooth Audio - Media player configuration options
+
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config BT_MPL
+	bool "Support for media player"
+	select BT_CCID
+	select EXPERIMENTAL
+	help
+	  Enables support for media player
+	  Note that the provided media player is a sample that only provides a
+	  mock-up with no actual media being played.
+	  For real media playback, the sample must be extended, hooked up to a
+	  real media player or replaced with a real media player.
+
+if BT_MPL
+
+config BT_MPL_MEDIA_PLAYER_NAME
+	string "Media Player Name"
+	default "Player0"
+	help
+	  Use this option to set the name of the media player.
+
+config BT_MPL_MEDIA_PLAYER_NAME_MAX
+	# TODO: Remove - having the name (above) should give the length
```

Comment:
Could we remove it in this PR? Seems pretty trivial

Reply:
Ditto for the ones below

Reply:
I have reconsidered (and removed the TODOs), as i see that these are actually needed.
For the GMCS (which we have), the name can change runtime, when the player changes.  
If that happens, the URL may also change.

(The reason I kept it out of this PR was that that would be a code change, while this PR deals with config changes.)